src/Maker/MakeMigration.php

```diff
@@ -88,6 +91,10 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
         }
         // end 2.x support
 
+        if ($input->hasOption('formatted') && null !== $input->getOption('formatted')) {
```

Comment:
There's a problem with this condition, for me it always `true` without having added the `--formatted` option. Here you just need to know if the option has been added, or not. So like this :

```suggestion
        if ($input->getOption('formatted')) {
```

Reply:
you right ;)